site/en/blog/chromium-chronicle-21/index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "The Chromium Chronicle #21: Chrome OS E2E UI Automation"
+description: >
+  Tast is a new UI library has been created that uses the Chrome a11y tree
+  to control the Chrome OS UI. The library enables developers to easily
+  create E2E tests against any visible UI surface.
```

Comment:
nit: Could this be changed to "End-to-end (E2E)"? This way, maybe other instances of E2E used later on can remain as they are, after the acronym is established.